addons/payment/views/payment_provider_views.xml

```diff
@@ -95,10 +95,24 @@
                         <page string="Fees" name="fees" attrs="{'invisible': [('support_fees', '=', False)]}">
                             <group name="payment_fees">
                                 <field name="fees_active"/>
-                                <field name="fees_dom_fixed" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_dom_var" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_int_fixed" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_int_var" attrs="{'invisible': [('fees_active', '=', False)]}"/>
+                                <field name="fees_dom_fixed"
+                                       widget="monetary"
+                                       style="max-width: 5rem"
+                                       options="{'currency_field': 'main_currency_id'}"
+                                       attrs="{'invisible': [('fees_active', '=', False)]}"/>
+                                <field name="fees_dom_var"
+                                       widget="percentage"
+                                       style="max-width: 5rem"
```

Comment:
I've seen this style being used elsewhere in Odoo, but, as we generally avoid using `style` altogether, I'd be surprised if there was no Odoo class to tackle this layout issue. Do you mind looking around and/or asking the JS team?

Reply:
Let's invoke @bouvyd here before he kills you in your sleep for doing that.

Reply:
well the diff has changed but i'd say you either need class oe_inline (not probable given the structure, but worth mentioning) or indeed to be in a group inside a group (though you have 3 levels now, which is weird)

Reply:
Thanks @bouvyd for your answer. I added a third level to avoid spending half of the page for a percentage field (and have the percent symbol being close to the float field). A quarter of the page is enough, that's why I added a third group. Is it ok or to be avoided? It's already better than the custom `style="max-width: 5rem"`.

Reply:
That's the result with only 2 groups. Not really a nice display. Is there anything to correct or add to the percent widget then?

![image](https://user-images.githubusercontent.com/112173839/206417701-62abeee0-1fcc-415d-ac42-6f854d7aecca.png)

Wasn't it the reason all currency symbols were moved to the left hand-side?

Reply:
it was, but whilst it's not weird to read currencies with the symbol in front; it'd be very weird to see `% 12.4` 

as i said you could always add the class oe_inline, but it's frankly not a big ux issue that justifies non-compliant views :p (i consider that oe_inline is compliant, group within group within group is not)

Reply:
I now totally agree ^^ And you totally answered my question about Odoo compliant. 
But now that I see the effect of oe_inline, I think we should improve/complete the documentation about it: 

> oe_inline
prevent the usual line break following fields

What about something like

> prevent the usual line break following fields and limit the span of the div / group / field